ffi/va_list.c

```diff
@@ -10,6 +10,6 @@ struct wrap {
 void create_va_list(struct wrap *w, ...) {
     va_list ap;
     va_start(ap, w->num_args);
```

Comment:
Change this to `va_start(ap, w);`. The second parameter is _not_ the count, but rather the first parameter before the rest of the arguments.